aten/src/ATen/native/quantized/decomposed/Quantize.cpp

```diff
@@ -0,0 +1,22 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <torch/library.h>
+
+namespace at {
+namespace native {
+Tensor decomposed_quantize_per_tensor(const Tensor& input, double scale, int64_t zero_point, ScalarType dtype) {
```

Comment:
would it be more forward proof to have `qmin` and `qmax` in the signature?

Reply:
yeah I was thinking about this as well, should I add it in this PR?

Reply:
I think that would be good, it might be annoying to change the signature later as I think there are tests that check for BC compatibility of signatures of functions in native_functions.yaml

Reply:
OK, sg, will update the PRs then